qiskit/providers/models/backendconfiguration.py

```diff
@@ -570,16 +584,77 @@ def acquire(self, qubit: int) -> AcquireChannel:
             raise BackendConfigurationError("Invalid index for {}-qubit systems.".format(qubit))
         return AcquireChannel(qubit)
 
-    def control(self, channel: int) -> ControlChannel:
+    def control(self, qubits: Iterable[int] = None,
+                channel: int = None) -> List[ControlChannel]:
         """
         Return the secondary drive channel for the given qubit -- typically utilized for
         controlling multiqubit interactions. This channel is derived from other channels.
 
+        Args:
+            qubits: Tuple or list of qubits of the form `(control_qubit, target_qubit)`.
+            channel: Deprecated.
+
+        Raises:
+            BackendConfigurationError: If the ``qubits`` is not a part of the system.
+
         Returns:
-            Qubit control channel.
+            List of control channel.
```

Comment:
```suggestion
            List of control channels.
```

Reply:
Fixed in [0da2877](https://github.com/Qiskit/qiskit-terra/pull/4097/commits/0da28775b838183eedd71d9157f20c08081e2833).